controllers/controllers.js

```diff
@@ -69,3 +72,13 @@ exports.getArticles = (req, res, next) => {
     .catch(next)
 }
 
+exports.postComment = (req, res, next) => {
+    const  article_id  = req.params.article_id
+    const  comment  = req.body
+    const username = req.body.username
+    Promise.all([checkArticleIdExists(article_id), checkUserExists(username), insertComment(comment, article_id)]).then(([, , comment]) => {
+        res.status(201).send(comment)
```

Comment:
You should be sending the comment back on the key of comment rather than directly on the response body